.prow.yaml

```diff
@@ -0,0 +1,14 @@
+presubmits:
+  GoogleCloudPlatform/blueprints:
+    - name: test
+      decorate: true
+      decorate_config:
+        timeout: 30m
+      spec:
+        serviceAccountName: prowjob-default-sa
```

Comment:
No need for this, it is applied as a default if `serviceAccountName` is unspecified.

Reply:
done